simulation/navigator_gazebo/test/test_sim_integration.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python
+import rospy
+import unittest
+from nav_msgs.msg import Odometry
+from sensor_msgs.msg import Image, CameraInfo, PointCloud2
+
+PKG = "navigator_gazebo"
+
+
+class TestSimIntegration(unittest.TestCase):
+
+    def __init__(self, *args):
+        super(TestSimIntegration, self).__init__(*args)
+        rospy.sleep(1)
+
+    def test_odom(self):
+        rospy.Subscriber("/odom", Odometry, self.verify_odom)
+
+    def verify_odom(self, data):
+        beg_pos_x = -1.2319
```

Comment:
Leave a comment saying why it's these numbers. Also suggest setting the near zero values to 0 as you are checking almost equal anyways